src/App.js

```diff
@@ -4,6 +4,17 @@ import PrivateRoute from "./PrivateRoute";
 import Auth0 from "./Auth0";
 import Home from "./views/Home";
 import OrgTopLevel from "./views/Organization";
+import { ApolloProvider } from '@apollo/react-hooks';
+import ApolloClient from 'apollo-boost';
+import { ApolloTest } from "./ApolloTest";
+
+const { 
+  REACT_APP_GRAPHQL_SERVER
+} = process.env;
```

Comment:
It would be good to update the example.env as well, so that other users know to update their .env files before getting stuck on this

Reply:
Done